lib/davinci_pas_test_kit/session_identification.rb

```diff
@@ -0,0 +1,45 @@
+require 'udap_security_test_kit'
+require_relative 'urls'
+
+module DaVinciPASTestKit
+  module SessionIdentification
+    def inputs_to_wait_identifier(client_id, session_url_path)
+      # look at test config and determine the wait identifier to use
+      # at somepoint this would be an inferno type, for now, just two options
+      return client_id if client_id.present?
+      return session_url_path if session_url_path.present?
+
+      test_session_id
+    end
+
+    def inputs_to_session_endpont(endpoint, client_id, session_url_path)
```

Comment:
I just didn't think it was clear what these methods did by their names
```suggestion
    def session_wait_identifier(client_id, session_url_path)
      # look at test config and determine the wait identifier to use
      # at somepoint this would be an inferno type, for now, just two options
      return client_id if client_id.present?
      return session_url_path if session_url_path.present?

      test_session_id
    end

    def session_endpont_url(endpoint, client_id, session_url_path)
```

Reply:
updated in [e11f366](https://github.com/inferno-framework/davinci-pas-test-kit/pull/27/commits/e11f36633d2602306926e8bacc307c71e69a336c)